client/src/components/Ticketing/ticketingmanager/Events/EventForm.tsx

```diff
@@ -243,21 +236,47 @@ const EventForm = ({onSubmit, tickettypes, initialValues}: EventFormProps) => {
                 <InputFieldForEvent
                   name={'eventname'}
                   id={'eventname'} headerText={'Enter Event Name'}
-                  action={addeventname} actionType={'onChange'} value={def.eventname}
-                  placeholder={def.eventname ? def.eventname: 'Event Name'} />
+                  action={addeventname} actionType={'onChange'} value={eventname}
+                  placeholder={'Event Name'} />
 
                 <InputFieldForEvent
                   name={'eventdescription'}
                   id={'eventdescription'} headerText={'Enter Short Event Description'}
                   actionType={'onChange'}
-                  action={addEventDesc} value={def.eventdescription}
-                  placeholder={def.eventdescription ? def.eventdescription : 'Event Description'} />
+                  action={addEventDesc} value={eventdescription}
+                  placeholder={'Event Description'} />
 
-                <InputFieldForEvent
-                  name={'imageurl'}
-                  id={'imageurl'} headerText={'Upload Image for Event'}
-                  action={addURL} actionType={'onChange'} value={def.imageurl}
-                  placeholder={def.imageurl ? def.imageurl : 'image URL'}/>
+                <div className={'grid grid-cols-5'}>
+                  <div className='col-span-3 md:col-span-4'>
+                    <InputFieldForEvent
+                      name={'imageurl'}
+                      id={'imageurl'} headerText={'Upload Image for Event'}
+                      action={addURL} actionType={'onChange'} value={imageurl}
+                      disabled={disabledUrl}
+                      placeholder={'image URL'}/>
+                  </div>
+                  <div className='col-span-2 md:col-span-1 flex flex-col mb-4'>
+                    <label
+                      htmlFor={'defaultImageUrl'}
+                      className={'text-sm text-zinc-600 text-center'}
+                    >
+                      Use Default Image
+                    </label>
+                    <input
+                      name={'defaultImageUrl'}
+                      id={'defaultImageUrl'}
+                      type='checkbox'
+                      value={'default'}
+                      checked={disabledUrl}
+                      onChange={ () => {
+                        setImageURL((!disabledUrl?'Default Event Image':''));
+                        setDisabledUrl(!disabledUrl);
+                      }}
+                      className={'appearance-none w-8 h-8 p-2 m-auto checked:bg-blue-500 ' +
```

Comment:
I think it would still be a good idea to have the checkmark exist, rather than only a "blue" filled color, so I don't think `appearance-none` would be ideal. I think this change would require adding a `rounded-...` class though.

Reply:
For consistency sake, it might be best just to use a normal checkbox like is used in other parts of the application
![image](https://github.com/WonderTix/WonderTix/assets/92124260/7ab37fb6-6f41-4689-aa5c-32eb13c9f134)

Reply:
Updated